From dd2abd5ce18359f757a0684f116a672fe0ed3e23 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Tue, 19 Nov 2024 13:23:52 +0800
Subject: [PATCH 280/360] x86: configs: enable AAEON drivers

Enable AAEON GPIO, HWMON, LEDs, and Multi-Function devices. These are
needed by Asus Vivobook S 2024 LED.

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 arch/x86/configs/deepin_x86_desktop_defconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/x86/configs/deepin_x86_desktop_defconfig b/arch/x86/configs/deepin_x86_desktop_defconfig
index bf1e89b619ea..af32e0dd9a91 100644
--- a/arch/x86/configs/deepin_x86_desktop_defconfig
+++ b/arch/x86/configs/deepin_x86_desktop_defconfig
@@ -2139,6 +2139,7 @@ CONFIG_GPIO_WHISKEY_COVE=m
 CONFIG_GPIO_WM831X=m
 CONFIG_GPIO_WM8350=m
 CONFIG_GPIO_WM8994=m
+CONFIG_GPIO_AAEON=m
 CONFIG_GPIO_AMD8111=m
 CONFIG_GPIO_MERRIFIELD=m
 CONFIG_GPIO_ML_IOH=m
@@ -2248,6 +2249,7 @@ CONFIG_CHARGER_WILCO=m
 CONFIG_BATTERY_SURFACE=m
 CONFIG_CHARGER_SURFACE=m
 CONFIG_BATTERY_UG3105=m
+CONFIG_SENSORS_AAEON=m
 CONFIG_SENSORS_ABITUGURU=m
 CONFIG_SENSORS_ABITUGURU3=m
 CONFIG_SENSORS_AD7314=m
@@ -4051,6 +4053,7 @@ CONFIG_LEDS_CLASS_FLASH=m
 CONFIG_LEDS_CLASS_MULTICOLOR=m
 CONFIG_LEDS_BRIGHTNESS_HW_CHANGED=y
 CONFIG_LEDS_88PM860X=m
+CONFIG_LEDS_AAEON=m
 CONFIG_LEDS_APU=m
 CONFIG_LEDS_LM3530=m
 CONFIG_LEDS_LM3532=m
-- 
2.20.1

